### DISCUSSION TITLE
Unable to enable PostgreSQL Protocol Credentials

### DISCUSSION THREAD

**Discussion by mugu-fatman** [ravendb/ravendb#14829] [Q&A]
Even though our license says PostgreSQL Protocol Support is enabled we are not able to configure it. Error says its an experimental feature and the server does not support experimental features. How to enable this feature in settings.json if we are using RavenDB cloud version. Please let me know if there is any solution to this issue. 

I have attached the license and error message pictures 
![image](https://user-images.githubusercontent.com/62667193/187244060-d93a4e42-04d8-4783-8f68-c5c88cf009e0.png)

![image](https://user-images.githubusercontent.com/62667193/187242772-c94beab0-8525-4645-9895-9a62737d02de.png)

**Comment by garayx** [comment]
Please submit a ticket using the support form, on [cloud portal](https://cloud.ravendb.net/portal), and ask to enable the experimental features on your instance.
